client/plots/matrix.renderers.js

```diff
@@ -324,16 +325,77 @@ export function setRenderers(self) {
 		const y = (s.collabelpos == 'top' ? d.mainh + s.collabelmaxchars : -s.collabelmaxchars) + 8
 		const anchor = s.rowlabelpos == 'left' ? 'end' : 'start'
 		const cl = s.controlLabels
-		const g = box.append('g').attr('class', 'sjpp-matrix-divide-by-label').attr('transform', `translate(0, ${y})`)
-		g.append('text')
+
+		const gNote = box.append('g').attr('class', 'sjpp-matrix-divide-by-label').attr('transform', `translate(0, ${y})`)
+
+		gNote
+			.append('text')
 			.attr('text-anchor', anchor)
 			.attr('font-style', 'italic')
 			.attr('y', -20)
 			.text(`${cl.Samples} grouped by`)
-		g.append('text').attr('text-anchor', anchor).attr('font-weight', 600).text(text)
+
+		gNote
+			.append('title')
+			.text(
+				`${cl.Samples} are grouped by this gene or variable. Use the Samples -> 'Group Samples By' input in the controls bar to edit.`
+			)
+
+		const g = box
+			.datum({ tw: self.config.divideBy })
+			.append('g')
+			.attr('class', 'sjpp-matrix-divide-by-label')
+			.attr('transform', `translate(0, ${y})`)
+
+		const textElem = g
+			.append('text')
+			.attr('text-anchor', anchor)
+			.attr('font-weight', 600)
+			.text(text)
+			.on('click', (event, d) => {
+				pill.showMenu(event, textElem.node())
```

Comment:
when the pill term is a categorical term with <= 2 categories e.g. Gender (count keys in term.values{}), then do not show Edit option, as is done when clicking term row label

Reply:
pnet doens't show Edit option when categorical term <= 2 categories, but gdc matrix show Edit option for gender when clicking term row label or when clicking the **Group Cases By term** inside of cases control ui 
<img width="673" alt="Screenshot 2023-09-20 at 17 21 15" src="https://github.com/stjude/proteinpaint/assets/111394589/d55e3dbe-8b0b-4766-914f-f90051d4606c">

Reply:
sorry didn't notice it's working for pnet. gdc is because the term.values{} lacks content. thanks!